lienol/luci-app-passwall/root/usr/share/passwall/app.sh

```diff
@@ -532,11 +549,10 @@ start_dns() {
 	dns2socks)
 		DNS2SOCKS_SOCKS_SERVER=$(echo $(config_t_get global socks_server nil) | sed "s/#/:/g")
 		[ "$DNS2SOCKS_SOCKS_SERVER" != "nil" ] && {
-			DNS2SOCKS_FORWARD=$(echo $DNS_FORWARD | awk -F ',' '{print $1}')
```

Comment:
or we prefer `#` spliter, DNS2SOCKS_FORWARD and pdnsd won't work with multi-value currently.